e2e/steps_file.js

```diff
@@ -169,6 +169,25 @@ module.exports = function () {
       }
     },
 
+    async fillDateAndTime(date, sectionId = 'form') {
+      if (date instanceof Date) {
+        date = {day: date.getDate(), month: date.getMonth() + 1, year: date.getFullYear(),
```

Comment:
Any reason why + 1 is done here? I guess it's because we use 0-based month in the setup file? couldn't we use just normal 1:1 month? seems a bit weird!

Reply:
Date data type uses 0 based month indexes, when we pass our custom date object then it is, or at least should be, 1 based index

Reply:
yes, its 0-based month - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth